llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1306,6 +1253,10 @@ void VPlanTransforms::addActiveLaneMask(
 /// ...
 ///
 void VPlanTransforms::addExplicitVectorLength(VPlan &Plan) {
+  VPValue *HeaderMask = getHeaderMask(Plan);
+  if (!HeaderMask)
```

Comment:
If addExplicitVectorLength() is called header mask is expected to exist - assert instead of early exit?